src/ng/compile.js

```diff
@@ -737,6 +737,11 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
 
     var bindings = {};
 
+    if (scope.hasOwnProperty !== Object.prototype.hasOwnProperty) {
```

Comment:
there is a function named `hasOwnProperty` that lives in the angular closure that should be used here